frontend/src/store/modules/notification/actions.js

```diff
@@ -0,0 +1,18 @@
+import { SHOW_SUCCESS_MESSAGE, SHOW_ERROR_MESSAGE, HIDE_MESSAGE } from './types/actions';
+import { SHOW, HIDE } from './types/mutations';
+
+export default {
+    [SHOW_SUCCESS_MESSAGE]: ({ commit }, text) => {
+        commit(SHOW, { text, type: 'success' });
+        setTimeout(() => commit(HIDE), 5000);
```

Comment:
Have you tried to use vuetify component to do this? It seems it already can do this for you